.circleci/config.yml

```
@@ -28,15 +28,16 @@ jobs:
           fingerprints:
               - "02:df:a5:6a:53:9a:f5:5d:bd:a6:fc:b2:db:9b:c9:47" # disable-secrets-detection
               - "f5:25:6a:e5:ac:4b:84:fb:60:54:14:82:f1:e9:6c:f9" # disable-secrets-detection
+      - run:
+          name: Create ID Set
+          command: |
+            python ./Tests/scripts/update_id_set.py -r
       - run:
           name: Infrastucture testing
           command: |
             pytest ./Tests/scripts/hook_validations/tests/ -v
             pytest ./Tests/scripts/infrastructure_tests/ -v
-      - run:
-          name: Create ID Set
-          command: |
-            python ./Tests/scripts/update_id_set.py -r
+            python ./Tests/scripts/test_configure_tests.py
```

```suggestion
            pytest ./Tests/scripts/test_configure_tests.py -v
```
Let's run the tests with pytest. The output is cleaner and easier to understand when there is a failure.